From 10fd75ffb980fd0d8edc3d7a886a4d797a9e740d Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Thu, 26 May 2022 11:39:00 +0300
Subject: [PATCH] [SW-71880] gaudi2: init cpu_dyn_regs with 2 more regs

There are 2 new registers that are added to the compatibility layer.
Initialize the relevant fields in the registers compat structure.

Change-Id: I9ac161025f697e2ee494941b9c4ade7f38f4bd70
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/arch/gaudi2/common/common.c b/arch/gaudi2/common/common.c
index 57fecd3..dc3fca3 100644
--- a/arch/gaudi2/common/common.c
+++ b/arch/gaudi2/common/common.c
@@ -197,6 +197,8 @@
 	regs->gic_host_halt_irq = mmGIC_HOST_HALT_IRQ_POLL_REG;
 	regs->gic_host_ints_irq  = mmGIC_HOST_INTS_IRQ_POLL_REG;
 	regs->gic_host_soft_rst_irq = mmGIC_HOST_SOFT_RST_IRQ_POLL_REG;
+	regs->cpu_rst_status = mmCPU_RST_STATUS_TO_HOST;
+	regs->eng_arc_irq_ctrl = mmENGINE_ARC_IRQ_CTRL_POLL_REG;
 }
 
 static struct hl_ts_cfg gaudi2_hl_ts_data[] = {
